app/controllers/api/v1/talent_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
   end
 
   def update
-    if talent.id != current_user.talent.id
+    if !current_user.admin? && talent.id != current_user.talent&.id
```

Comment:
Should it be or here?

```suggestion
    if !current_user.admin? || talent.id != current_user.talent&.id
```